From a11946f7663ee4da3809ff1be911f2415d2ea835 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 25 Jan 2013 19:54:21 +0100
Subject: [PATCH 178/204] job: fix merging with --ignore-dependencies

This fixes a bug where a job with --ignore-dependencies would wait for
other jobs because it merged into a previously queued job.
(cherry picked from commit e45460d666512db4f908f86e8722d7932dcf0f82)
---
 src/core/job.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/core/job.c b/src/core/job.c
index f08b8cb..27da082 100644
--- a/src/core/job.c
+++ b/src/core/job.c
@@ -166,6 +166,7 @@ static void job_merge_into_installed(Job *j, Job *other) {
                 assert(other->type == JOB_NOP);
 
         j->override = j->override || other->override;
+        j->ignore_order = j->ignore_order || other->ignore_order;
 }
 
 Job* job_install(Job *j) {
-- 
1.8.1.5

